app/gui2/src/composables/navigator.ts

```diff
@@ -258,6 +271,7 @@ export function useNavigator(viewportNode: Ref<Element | undefined>, keyboard: K
 
   return proxyRefs({
     events: {
+      ...panArrows.events,
```

Comment:
Not sure if it's good idea to spread those. The returned `events` are all bound with `v-on` to the wrapping `div`, which probably is unintentional here.